res/menu/event_info_title_bar.xml

```diff
@@ -20,14 +20,14 @@
         android:alphabeticShortcut="s"
         android:title="@string/share_label"
         android:icon="@drawable/ic_share_event"
-        app:showAsAction="always" />
+        app:showAsAction="ifRoom" />
```

Comment:
why this? Dito the two below.

Reply:
They are not often used, and create a cluttered ui. We should show only often used and important actions, and hide the rest. Showing 4 or more icons in the topbar is to much and unnessesssary.